### ISSUE TITLE
dropdown lists are not accessible

### ISSUE THREAD

**Issue by desobryc** [swiss/styleguide#256]
![image](https://cloud.githubusercontent.com/assets/7644740/4441381/42a02e3a-47d2-11e4-881a-6cb208f8b85e.png)
